pkg/webui/account/views/front/index.js

```diff
@@ -43,6 +47,11 @@ const FrontView = () => {
             <Route path="/validate" component={Validate} />
             <Route path="/code" component={Code} />
             <Redirect exact from="/" to="/login" />
+            {authRoutes.map(({ path, exact }) => (
+              <Route path={path} exact={exact} key={path}>
+                <Redirect to={`/login?n=${selectApplicationRootPath()}${location.pathname}`} />
```

Comment:
What is the purpose of this redirect?

Reply:
The purpose is to redirect routes that require authentication back to the login page, with a query parameter that redirectes to the original route after login. E.g. `/profile-settings` (will be added later) would redirect back to the login page. All other routes that do not correspond to anything will show a 404.

Note that the `<Front />` handler only operates when the user is not authenticated.